src/category_theory/limits/comma.lean

```diff
@@ -205,6 +207,14 @@ noncomputable instance creates_limits_of_shape [preserves_limits_of_shape J G] :
 noncomputable instance creates_limits [preserves_limits G] :
   creates_limits (proj X G : _) := ⟨⟩
 
+instance mono_right_of_mono [has_limits A] [preserves_limits G] {Y Z : structured_arrow X G}
+  (f : Y ⟶ Z) [mono f] : mono f.right :=
```

Comment:
We get an analogous result for the over/under categories from this right?

Reply:
Yes, I added this.